portal-api/src/modules/services/enrollment/enrollment.app.ts

```diff
@@ -0,0 +1,133 @@
+import { v4 as uuidv4 } from 'uuid';
+import {
+  CanEnrollResponse,
+  CanEnrollStatus,
+  OrganizationCapability,
+  ServiceDefinitionIdentifier,
+} from '../../../__generated__/resolvers-types';
+import { PortalContext } from '../../../model/portal-context';
+import { isUserAllowed } from '../../../security/auth.helper';
+import { loadSubscriptionByServiceInstance } from '../../subcription/subscription.domain';
+import { serviceContractDomain } from '../contract/domain';
+import { serviceDefinitionDomain } from '../definition/domain';
+import {
+  enrollmentDomain,
+  OCTIInstanceConfiguration,
+} from './enrollment.domain';
+
+interface EnrollOCTIInstancePayload {
+  organizationId: string;
+  platformId: string;
+  platformUrl: string;
+  platformTitle: string;
+}
+
+export const enrollmentApp = {
+  enrollOCTIInstance: async (
+    context: PortalContext,
+    {
+      organizationId,
+      platformId,
+      platformUrl,
+      platformTitle,
+    }: EnrollOCTIInstancePayload
+  ): Promise<string> => {
+    const token = uuidv4();
+    const configuration: OCTIInstanceConfiguration = {
+      enroller_id: context.user.id,
+      platform_id: platformId,
+      platform_url: platformUrl,
+      platform_title: platformTitle,
+      token: token,
+    };
```

Comment:
```suggestion
export const enrollmentApp = {
  enrollOCTIInstance: async (
    context: PortalContext,
    {
      organizationId,
      platformId,
      platformUrl,
      platformTitle,
    }: EnrollOctiInstanceInput
  ): Promise<string> => {
    const token = uuidv4();
    const configuration: OCTIInstanceConfiguration = {
      enroller_id: context.user.id,
      platform_id: platformId,
      platform_url: platformUrl,
      platform_title: platformTitle,
      token: token,
    };
```
For the payload, better use the one you get from the graphql definition `EnrollOctiInstanceInput`